tests/test_backend.py

```diff
@@ -1410,6 +1410,20 @@ def test_sparse_softmax_cross_entropy_with_logits(self):
 
         self._run_test_case([_OUTPUT], {_INPUT: label_val, _INPUT1: logits_val})
 
+    @unittest.skipIf(*support_op_with_target('rs6', 'SparseSoftmaxCrossEntropyWithLogits'))
```

Comment:
on a second thought, i feel it is kind of ambiguous to use 'rs6' , for both converter code and test.
what we really want is when backend is onnxruntime and its version equals to xxx, do some special hacks in conversion.

we may introduce "onnxruntime" as a target, and enhance current target handling logic to support version (optional).
e.g. 
--target onnxruntime
--target onnxruntime:0.1.4

and in test, we can skip if backend==onnxruntime and target != onnxruntime.

Reply:
for this pr, just keep it this way.
we can create new pr if we want to add onnxruntime as target.

Reply:
@pengwa to comment